### DISCUSSION TITLE
File name as header is gone

### DISCUSSION THREAD

**Discussion by embamba** [chrisgrieser/shimmering-focus#309] [Q&A] [answered]
It seems that with the new version of Shimmering Focus, 4.1, the file name header in the workspace is gone. Now, when creating a new file, I get a dialog asking for a name for the new file. I don't prefer this. I like the old way with the editable file name at the top of the workspace. (I hope my terminology is correct.) Other themes seem to still do it the old way and I've only seen this issue since updating the theme, so I'm guessing it's theme related. I don't see anything in Style Settings that allows me to revert this. 

What can be done to revert this for me? Or is it now baked into the theme?

**Comment by chrisgrieser** [comment, marked as answer]
Not totally sure what header exactly you mean. In case you mean in the inline title, I just pushed an update that fixes this.

If this is not the case, please add screenshots, since it's hard to guess what you mean otherwise. Also, the bug report form would also be useful, since it asks for things like screenshots.

**Comment by embamba** [comment]
Yes, it was the inline title and 4.5 fixed it. 

Thanks for responding and for the pointers.
